schema.ts

```diff
@@ -834,6 +834,37 @@ export interface ObjectSchemaDefinition<K extends string, L extends string> exte
    * scenarios they can be useful.
    */
   includeUnknownProperties?: boolean;
+  /**
+   * The name of a property within {@link properties} that can be used to attribute the source
+   * of this object in the UI.
+   *
+   * Must be a string property with a URL hint.
+   */
+  /** @hidden */
+  linkProperty?: L;
+  /**
+   * A list of property names from within {@link properties} for the properties of the object
+   * to be shown in the subtitle of a rich card preview for formulas that return this object.
+   * Defaults to the value of {@link featuredProperties} if not specified.
+   */
+  /** @hidden */
+  subtitleProperties?: L[];
```

Comment:
If we're punting on `titleProperty` until we find a case where `displayProperty` doesn't work, is it worth doing the same here? Or do we see a need for this immediately?

Reply:
ah sorry this is a good catch I meant to include that at first for us to play with and then decide to remove later per the slack thread